lib/dom-patcher.js

```diff
@@ -93,6 +94,16 @@ export class DOMPatcher {
     patch(this.vnode, newVnode);
     this.vnode = newVnode;
     this.el = this.vnode.elm;
+    const elapsedMs = Date.now() - startedAt;
+    if (elapsedMs > this.slowThreshold) {
```

Comment:
This code is in DOMPatcher, but `slowThreshold` is on Component, isn't it?

Reply:
yeah, good call 🤦